common/src/main/java/com/box/l10n/mojito/github/GithubClient.java

```diff
@@ -86,6 +86,22 @@ public void addCommentToPR(String repository, int prNumber, String comment) {
     }
   }
 
+  public void addCommentToCommit(String repository, String commit, String comment) {
```

Comment:
Nitpick but maybe update the `commit` variable name to `commitSha` for clarity?

Reply:
good one thanks!